lib/helpers/TemplateUiHelper.ts

```diff
@@ -0,0 +1,132 @@
+﻿import {Page, Locator} from "@playwright/test"
+
+export class TemplateUiHelper {
+  private readonly page: Page;
+  private readonly newEmptyTemplateBtn: Locator;
+  private readonly insertTemplateName: Locator;
+  private readonly saveBtn: Locator;
+  private readonly deleteFolderBtn: Locator;
+  private readonly caretBtn: Locator;
+  private readonly templateTextArea: Locator;
+  private readonly deleteBtn: Locator;
+  private readonly confirmToDeleteBtn: Locator;
+  private readonly insertFolderName: Locator;
+  private readonly createFolderBtn: Locator;
+  private readonly changeMasterTemplateBtn: Locator;
+  private readonly queryBuilderBtn: Locator;
+  private readonly queryBuilderOrderedByBtn: Locator;
+  private readonly queryBuilderCreateDate: Locator;
+  private readonly submitBtn: Locator;
+  private readonly clickSectionsBtn: Locator;
+  private readonly clickInsertValueBtn: Locator;
+  private readonly insertDictionaryItemBtn: Locator;
+
+  constructor(page: Page) {
+    this.page = page;
+    this.newEmptyTemplateBtn = page.getByLabel('Create');
+    this.insertTemplateName = page.getByLabel('template name');
+    this.saveBtn = page.getByLabel('Save');
+    this.submitBtn = page.getByLabel('Submit');
+    this.caretBtn = page.locator('div').filter({hasText: 'Templates'}).locator('#caret-button')
+    this.templateTextArea = page.locator('textarea.inputarea')
+    this.deleteBtn = page.getByRole('button', {name: 'Delete'});
+    this.confirmToDeleteBtn = page.locator('#confirm').getByLabel('Delete');
+    this.createFolderBtn = page.getByLabel('Create folder');
+    this.insertFolderName = page.locator('[headline="Create Folder"] >> input');
+    this.deleteFolderBtn = page.getByLabel('Delete');
+    this.changeMasterTemplateBtn = page.getByLabel('Change master template');
+    this.queryBuilderBtn = page.locator('#query-builder-button').getByLabel('Query builder')
+    this.queryBuilderOrderedByBtn = page.locator('#property-alias-dropdown').getByLabel('Property alias');
+    this.queryBuilderCreateDate = page.locator('#property-alias-dropdown').getByText('CreateDate')
+    this.clickSectionsBtn = page.locator('#sections-button', {hasText: 'Sections'})
+    this.clickInsertValueBtn = page.getByLabel('Choose value to insert');
+    this.insertDictionaryItemBtn = page.getByLabel('Insert Dictionary item');
+  }
+
+  async openActionsMenuForName(name: string) {
+    await this.page.locator('[label="' + name + '"] >> [label="Open actions menu"]').click({force: true});
+  }
```

Comment:
This same method is also in the script helper, I think this should be in a base class or similar 😄